charts/incubator/odoo/values.yaml

```diff
@@ -46,15 +59,18 @@ persistence:
   odoo:
     enabled: true
     mountPath: "/var/lib/odoo"
+    targetSelectAll: true
   addons:
     enabled: true
     mountPath: "/mnt/extra-addons"
+    targetSelectAll: true
   odoo-config:
     enabled: true
     type: configmap
     objectName: odoo-config
     mountPath: /etc/odoo/odoo.conf
     subPath: odoo.conf
+    targetSelectAll: true
```

Comment:
Please use targeted mounts, there is no need for select all :)
See nextcloud for example